open-banking-accelerator/components/com.wso2.openbanking.accelerator.common/src/main/java/com/wso2/openbanking/accelerator/common/util/JWTUtils.java

```diff
@@ -120,4 +137,105 @@ public static boolean validateJWTSignature(String jwtString, String jwksUri, Str
         return true;
     }
 
+    /**
+     * Validates the signature of a given JWT against a given public key.
+     *
+     * @param signedJWT the signed JWT to be validated
+     * @param publicKey the public key that is used for validation
+     * @return true if signature is valid else false
+     * @throws NoSuchAlgorithmException if the given algorithm doesn't exist
+     * @throws InvalidKeySpecException if the provided key is invalid
+     * @throws JOSEException if an error occurs during the signature validation process
+     */
+    @Generated(message = "Excluding from code coverage as KeyFactory does not initialize")
+    public static boolean validateJWTSignature(SignedJWT signedJWT, String publicKey)
```

Comment:
For methods returning a boolean value, it's a good practice to name them as questions. 
Examples: `StringUtils.isEmpty()` `Optional.isPresent()`. Please check other places too

```suggestion
    public static boolean isSignatureVerified(SignedJWT signedJWT, String publicKey)
```

Reply:
Resolved in [83e631f](https://github.com/wso2/financial-open-banking/pull/67/commits/83e631f5cf7f0e52f6460082da104e93ca98e07e).